qemu/tests/blockdev_snapshot_guest_agent.py

```diff
@@ -0,0 +1,69 @@
+import logging
+import time
+import random
```

Comment:
we can remove it for random is never used

Reply:
Have updated, thanks zhenchao for pointing it out.